File: C/1155.c
#include <stdio.h>
int main()
{
    double i, b = 1, c, soma = 0;
    for(i = 1; i < 101; i++)
    {
        c = 1 / i;
        soma += c;
    }
    printf("%.2lf\n", soma);
    return 0;
}

File: Java/1155.java
import java.io.IOException;

public class Main {
    public static void main(String[] args) throws IOException {
        double b = 1, c, soma = 0;
        for(double i = 1; i < 101; i++)
        {
            c = 1 / i;
            soma += c;
        }
        System.out.printf("%.2f\n", soma);
    }
}